android/build.gradle

```diff
@@ -1,17 +1,17 @@
 buildscript {
-    ext.kotlin_version = '1.7.20'
+    ext.kotlin_version = '1.7.10'
```

Comment:
Why Kotlin version down?

Reply:
I might have made a typing mistake in this. I will correct it now